explorer/explorerroutes.go

```diff
@@ -241,7 +245,15 @@ func (exp *explorerUI) TxPage(w http.ResponseWriter, r *http.Request) {
 				} else if tx.TicketInfo.VoteLuck == 0 {
 					tx.TicketInfo.LuckStatus = "No Luck"
 				}
-				tx.TicketInfo.Probability = (1 - math.Pow((1-(float64(exp.ChainParams.TicketsPerBlock)/float64(exp.ExtraInfo.PoolInfo.Size))), (float64(exp.ChainParams.TicketExpiry)-float64(tx.Confirmations)-float64(exp.ChainParams.TicketMaturity)))) * 100
+
+				// Chance for a ticket to NOT be voted in a given time frame:
+				// C = (1 - P)^N
+
+				// Where:
+				// P is the probability of a vote in one block. (votes per block / current ticket pool size)
+				// N is the number of blocks before ticket expiry. (ticket expiry in blocks - (number of blocks since ticket purchase - ticket maturity))
+				// C is the probability (chance)
+				tx.TicketInfo.Probability = (math.Pow((1 - (float64(exp.ChainParams.TicketsPerBlock) / float64(exp.ExtraInfo.PoolInfo.Size))), (float64(exp.ChainParams.TicketExpiry) - (float64(tx.Confirmations) - float64(exp.ChainParams.TicketMaturity))))) * 100
```

Comment:
By splitting this up, it would be easier to follow.

e.g.:
```
blocksLive := float64(tx.Confirmations) - float64(exp.ChainParams.TicketMaturity)
pVote := float64(exp.ChainParams.TicketsPerBlock) / float64(exp.ExtraInfo.PoolInfo.Size)
// or just pVote := 1 / exp.ChainParams.TicketPoolSize
...